src/main/java/uk/gov/companieshouse/authcodenotification/model/SendEmailRequestDto.java

```diff
@@ -4,7 +4,9 @@
 
 public class SendEmailRequestDto {
 
-    @JsonProperty("auth_code")
+    public static final String AUTH_CODE_FIELD = "auth_code";
```

Comment:
Not sure we need to change this. I would just define an `AUTH_CODE_PARAMETER` constant as you did for company number in `AuthCodeEmailValidator`.

Reply:
Moved this to the validator